### PR TITLE
Added test cases related to WPE-464 task in JIRA

### PR DESCRIPTION
**Pull request by sandeep-vedam**
Added test cases related to WPE-464 : 
* FrameworkHeaderCache001
* FrameworkHeaderCache002
* commonFunctions

### CODE REVIEW

File: src/framework/Framework_Headers_Cache_001.test.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  getResponseForURLRequest,
+  restartFramework,
+} from '../commonMethods/commonFunctions'
+import constants from '../commonMethods/constants'
+
+let count = 0
+export default {
+  title: 'Validate cache headers on all plugins',
+  description: 'Iterate over all plugins and verify their cache headers when activated',
+  teardown: restartFramework,
```
Comment (wouterlucas) [inline src/framework/Framework_Headers_Cache_001.test.js:12]: No need

File: src/commonMethods/commonFunctions.js [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+}
+
+/**
+ * This function is used to put request
+ * @param plugin
+ * @returns {Promise<AxiosResponse<any>>}
+ */
+export const putRequestForPlugin = function() {
+  return this.$http
+    .put(`http://${constants.host}:80/Service/Provisioning`)
```
Comment (wouterlucas) [inline src/commonMethods/commonFunctions.js:349]: REST APIs are deprecated, should be done using JSONRPC. If API is not available in JSONRPC yet we should not be making test cases for it yet, instead track it somewhere as "not supported on JSONRPC" in a ticket.

File: src/commonMethods/commonFunctions.js [hunk 3]
```diff
@@ -0,0 +1,397 @@
...
+    .then(result => result)
+    .catch(err => err)
+}
+
+/**
+ * This function is used to perform GET request operation on the URL
+ * @returns {Promise<AxiosResponse<any>>}
+ */
+export const getReqURL = function(URL) {
+  return this.$http
```
Comment (wouterlucas) [inline src/commonMethods/commonFunctions.js:359]: Same here, REST APIs are deprecated.

File: src/commonMethods/commonFunctions.js [hunk 4]
```diff
@@ -0,0 +1,397 @@
...
+    return putReqURL.call(this, url)
+  }
+}
+
+/**
+ * This function is used to perform {PUT} request operation on the URL
+ * @returns {Promise<AxiosResponse<any>>}
+ */
+export const putReqURL = function(URL) {
+  return this.$http
```
Comment (wouterlucas) [inline src/commonMethods/commonFunctions.js:384]: Same here, REST APIs are deprecated.

File: src/framework/Framework_Headers_Cache_002.test.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
+import {
+  getPluginInfo,
+  getResponseForURLRequest,
+  restartFramework,
+} from '../commonMethods/commonFunctions'
+import constants from '../commonMethods/constants'
+
+let count = 0
+export default {
+  title: 'Validate cache headers on all plugins',
```
Comment (wouterlucas) [inline src/framework/Framework_Headers_Cache_002.test.js:10]: This test validates the REST APIs of the Thunder stack. REST APIs are deprecated and I think we should not be testing for them anymore. Thoughts @Divya-563 ?

**Review by wouterlucas — CHANGES_REQUESTED** [review]

File: src/commonMethods/commonFunctions.js (hunk 2 above)
Comment (sandeep-vedam) [reply src/commonMethods/commonFunctions.js:349]: I could not find the location of JSONRPC for Provisioning Plugin. Please let me know the location of JSONRPC

File: src/framework/Framework_Headers_Cache_002.test.js (hunk 5 above)
Comment (sandeep-vedam) [reply src/framework/Framework_Headers_Cache_002.test.js:10]: Hello @Divya-563 , @wouterlucas ,
Please let me know whether this test case can be dropped .

Comment (wouterlucas) [reply src/framework/Framework_Headers_Cache_002.test.js:10]: Yes this can be dropped.

**Comment by sandeep-vedam** [comment]
As this PR is not needed, it is closed.

**Pull request closed without merging**